packages/client/src/fetcher.ts

```diff
@@ -43,14 +43,13 @@ export abstract class BaseFetcher implements Fetcher {
       // the length of the file as we go
       try {
         for await (const chunk of reader) {
-          const bufferChunk = Buffer.from(chunk);
-          numberOfBytesReceived += bufferChunk.length;
+          numberOfBytesReceived += chunk.length;
```

Comment:
Something in the `@types/node` update was not compatible with what we were doing here. Eliminating the use of `Buffer.from` seems to have solved the issue.